### ISSUE TITLE
null safety

### ISSUE THREAD

**Issue by anovis** [BMEC/dart-geo_hash_xy#1]
Thanks for the fork. I was wondering if you had plans to migrate this library to null safety?

https://dart.dev/null-safety/migration-guide

**Comment by isaacsuazo7** [comment]
Hi, I was also wondering if you have plans to migrate this library to null security.

**Comment by isaacsuazo7** [comment]
Hi, I was also wondering if you had plans to migrate this library to null safety?

**Comment by anovis** [comment]
@isaacsuazo7 I ended up migrating and merging the dart_geohash library into my package. 
https://pub.dev/packages/proximity_hash

**Comment by isaacsuazo7** [comment]
> @isaacsuazo7 I ended up migrating and merging the dart_geohash library into my package.
> https://pub.dev/packages/proximity_hash
@anovis does the package have the same functionalities? I use it to convert a LatLng into geohash. But still, I'll take a look!

**Comment by anovis** [comment]
Yeah. I copied over the full dart_geohash library so you will have access to all of the same methods.

**Comment by raphael-bmec-co** [comment]
Hi All

Apologies, this was not being maintained or monitored. We use https://pub.dev/packages/dart_geohash now. We will be deleting this branch.
